spec/lib/api/2.0/notification-spec.js

```diff
@@ -265,4 +271,75 @@ describe('Http.Api.Notification', function () {
             });
         });
     });
+
+    describe('POST /notification/alerts', function () {
+        var bmc = {
+                'IP Address': '10.240.19.130',
+                node : "abc",
+                'MAC Address':"jill"
+        };
+        var deviceInfoCatalog ={
+            data : {
+                'Builtin FRU Device (ID 0)' : {
+                    'Board Product' : "bernie",
+                    'Product Serial' : "hillary",
+                    'Board Serial' : "theDonald"
+                }
+            }
+        };
+
+
+        var alert = {
+            '{"Severity":"Critical"}': "d"};
```

Comment:
This is  how the body-parser is decoding the x-www-form-urlencoded data. 
The content or data is the in the `key` not the `value`. 
We have a story to look into the root cause of this https://rackhd.atlassian.net/browse/RAC-4699